data/harvard/law.free.cap.conn.254/902.1116733.json

```diff
@@ -0,0 +1,6 @@
+{
+  "casebody": {
+    "status": "ok",
+    "data": "<html><body><casebody firstpage=\"902\" lastpage=\"902\" xmlns=\"http://nrs.harvard.edu/urn-3:HLS.Libr.US_Case_Law.Schema.Case_Body:v1\">\n<parties id=\"b888-4\">MAURA ALEXANDER ET AL. <em>v. </em>RETIREMENT BOARD OF THE CITY OF WATERBURY</parties>\n\n<attorneys id=\"b888-7\"><em>Elena Ricci Palermo, </em>corporation counsel, in support of the petition.</attorneys>\n<attorneys id=\"b888-8\"><em>Henry F. Murray </em>and <em>Daniel E. Livingston, </em>in opposition.</attorneys>\n<decisiondate id=\"b888-9\">Decided July 6, 2000</decisiondate>\n<opinion><p id=\"b888-6\">SULLIVAN and VERTEFEUILLE, Js., did not participate in the consideration or decision of this petition.</p></opinion></casebody>\n</body></html>"
```

Comment:
this looks like it didnt work.

Reply:
I inspected the dataset and there are many examples with "did not participate in the consideration of or decision on this petition." in "opinion" category